### PR TITLE
fix(aio): temporarily remove link to source from the API pages

### LINKED ISSUE #17254: aio: odd formatting of module info on API page
While all other text on API pages is left justified, the module/package info is centered within an outline box that is itself too faint to read on many screens:

image

This seems to be easy to fix before release. Could punt if necessary.

### PR DESCRIPTION
**Pull request by gkalpak**
We need to come up with a better design (possibly involving an icon button) to link to the source code (for viewing and/or editing).

Fixes #17254.

### CODE REVIEW

File: aio/tools/transforms/templates/api/includes/info-bar.html [hunk 1]
```diff
@@ -11,8 +11,4 @@
     NgModule: {@link {$ doc.ngModule $}}
   </span>
   {% endif %}
```
Comment (gkalpak) [inline aio/tools/transforms/templates/api/includes/info-bar.html]: There is also this `NgModule`  entry that I didn't know what to do with. I also couldn't find a doc that has it (from a super quick look). @petebacondarwin, any idea what this is and if we need it?

**Comment by gkalpak** [comment]
Created #17372 to track adding a better link to the source.

File: aio/tools/transforms/templates/api/includes/info-bar.html (hunk 1 above)
Comment (petebacondarwin) [inline aio/tools/transforms/templates/api/includes/info-bar.html]: Here is a page that has this: https://aio-staging.firebaseapp.com/api/common/AsyncPipe

Comment (petebacondarwin) [inline aio/tools/transforms/templates/api/includes/info-bar.html]: And this is what it looks like now: https://pr17371-acca997053fbbe9e0a1f09a02a309d49d6877926.ngbuilds.io/api/common/AsyncPipe

Comment (petebacondarwin) [inline aio/tools/transforms/templates/api/includes/info-bar.html]: I think we should remove this too; and then put the NgModule in a section of its own.

**Comment by petebacondarwin** [comment]
There is an e2e test for the API docs link to github, which is now failing: https://travis-ci.org/angular/angular/jobs/241315553#L1687

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Green now!

**Pull request merged**